apps/acceptance/clock/.jshintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Hey Johan. With this one, I put it in acceptance because back when we decided to implement these tests, I was advised not to put them into the app structure until everyone was on board with the pattern and such. So I put them in their own folder until we got the go ahead to merge them into the app suites. End to end or e2e may definitely be a better name for the folder structure though.